assets/src/wizards/subscriptions/index.js

```diff
@@ -77,6 +78,8 @@ class SubscriptionsWizard extends Component {
 						disabled
 					/>
 				</Card>
+
+				<NewspackButton isPrimary value="Continue" />
```

Comment:
Per comment above about using `children` rather than `value`, this would change to:

```suggestion
				<NewspackButton isPrimary>Continue</NewspackButton>
```